pkg/virt-handler/vm_test.go

```diff
@@ -1906,6 +1906,56 @@ var _ = Describe("VirtualMachineInstance", func() {
 				err := controller.checkNetworkInterfacesForMigration(vmi)
 				Expect(err).To(HaveOccurred())
 			})
+			It("should block migration for VMI with SRIOV interface when feature-gate SRIOVLiveMigration is off", func() {
+				vmi := v1.NewMinimalVMI("testvmi")
+				sriovInterfaceName := "sriovnet1"
+				vmi.Spec.Networks = []v1.Network{
+					{
+						Name: sriovInterfaceName,
+						NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{
+							NetworkName: "sriov-network1",
+						}},
+					},
+				}
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					{
+						Name: sriovInterfaceName,
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							SRIOV: &v1.InterfaceSRIOV{},
+						},
+					},
+				}
+
+				Expect(controller.checkNetworkInterfacesForMigration(vmi)).ShouldNot(Succeed())
+			})
+
+			It("should not block migration for VMI with SRIOV interface when feature-gate SRIOVLiveMigration is on", func() {
+				vmi := v1.NewMinimalVMI("testvmi")
+				sriovInterfaceName := "sriovnet1"
+				kubevirtConfigMapFeatureGate := map[string]string{"feature-gates": "SRIOVLiveMigration"}
```

Comment:
Do we have constants for these keywords? ("feature-gates", "SRIOVLiveMigration")

Reply:
Nope, there is no other test in this package that handles feature-gate, you prefer to have constants for those in this file anyway?

Reply:
Done